logstreams/src/main/java/io/zeebe/distributedlog/restore/log/impl/DefaultLogReplicationServerHandler.java

```diff
@@ -81,6 +81,20 @@ public final LogReplicationResponse onReplicationRequest(LogReplicationRequest r
     return response;
   }
 
+  private boolean seekToRequestedPosition(long position, boolean includeFromPosition) {
```

Comment:
I don't think it's necessary to have two methods here just to avoid one branch condition. It would make sense in a more FP approach but here I'd just combine both methods:

```java
private boolean seekToRequestedPosition(long position, boolean skipToNextEvent) {
  if (position == -1) {
    reader.seekToFirstEvent();
  }

  if (reader.seek(position) && reader.hasNext()) {
    if (skipToNextEvent) {
      reader.next();
    }
    return true;
  }

  return false;
}
```

I know that the "exclusive" semantics are a bit different though, we can discuss that.

Reply:
I agree. Will update it.